common/azure-common/src/main/java/com/azure/common/implementation/SwaggerMethodParser.java

```diff
@@ -48,6 +46,7 @@
  * method.
  */
 public class SwaggerMethodParser implements HttpResponseDecodeData {
+    private static final int FALLTHROUGH_EXCEPTION_CODE = -1;
```

Comment:
This isn't a good approach. If a match isn't found in a map, just fall through - don't put something arbitrary into a map just to then look it up in a situation you know is exceptional.

Reply:
Added a defaultException member variable that will be used in place of this.